GeoHealthCheck/config_main.py

```diff
@@ -47,6 +47,7 @@
 GHC_NOTIFICATIONS_EMAIL = ['you2@example.com']
 GHC_SITE_TITLE = 'GeoHealthCheck Demonstration'
 GHC_SITE_URL = 'http://host'
+GHC_METADATA_CACHE_SECS = 7200
```

Comment:
Is more an advanced Python thing: maybe APP is initialized only once...see e.g.
https://stackoverflow.com/questions/31875/is-there-a-simple-elegant-way-to-define-singletons
APP may be already a Singleton, bound to its module once? (Was too long into Java where we explicitly defined Singletons). Do you know?